src/configurationProvider.ts

```diff
@@ -227,17 +229,17 @@ export class JavaDebugConfigurationProvider implements vscode.DebugConfiguration
                     return undefined;
                 }
                 if (!config.mainClass) {
-                    progressReporter.report("Resolve mainClass", "Resolving main class...");
+                    progressReporter.report("Resolving mainClass...");
```

Comment:
`Resolving main class...`

And don't forget to update all the other appearance for this message